indexing-service/src/main/java/io/druid/indexing/common/index/RealtimeAppenderatorTuningConfig.java

```diff
@@ -37,8 +37,9 @@
 @JsonTypeName("realtime_appenderator")
 public class RealtimeAppenderatorTuningConfig implements TuningConfig, AppenderatorConfig
 {
-  private static final int defaultMaxRowsInMemory = 75000;
+  private static final int defaultMaxRowsInMemory = 1000000;
   private static final int defaultMaxRowsPerSegment = 5_000_000;
+  private static final long defaultMaxBytesInMemory = getDefaultMaxBytesInMemory();
```

Comment:
This should use `IndexTaskUtils.DEFAULT_MAX_BYTES_IN_MEMORY` too.

Reply:
will change to `TuningConfig.DEFAULT_MAX_BYTES_IN_MEMORY` now.